views/page/course/audience/add-event.html

```diff
@@ -51,7 +51,7 @@ <h2 class="govuk-heading-m govuk-!-font-size-36">Assign event to a private audie
                     </div>
                 </form>
             {% else %}
-                <p class="govuk-body">There are no events to display</p>
+                <p class="govuk-body">There are no events - please add events to the course.</p>
```

Comment:
Where did this message come from?

Reply:
LPFG-735 AC: Give that I select to add an associated event
When there are no events on to select from
Then I receive an error message stating that I need to create an event

Reply:
I would follow up with Abi/Joel on this one.

Technically it's not an _error_ message that we are showing here. I would also want double check the contents of the message. I know the message isn't specified on the ticket, but it might be confusing to the user to say "_please add events..._" when there is no option to do so on this screen.